src/lib/lumi.ts

```diff
@@ -2045,17 +2045,24 @@ export const lumiModernExtend = {
             ...args,
         }),
     lumiVibration: (): ModernExtend => {
-        const exposes: Expose[] = [e.action(['shake', 'triple_strike'])];
+        const exposes: Expose[] = [e.action(['triple_strike', 'movement'])];
```

Comment:
Please keep the actions the same to prevent a breaking change.

Reply:
Oh, I was just following the naming in the Aqara app. If it's not appropriate, I'll revert back to the original action.
<img width="600" alt="aqara app" src="https://github.com/user-attachments/assets/aeeef001-112e-4667-858c-6089d833d3ac" />

Reply:
@Koenkk Hi, Kk ,What do you think?

Reply:
I would not go for a breaking change here, so please revert.

Reply:
Since I'm not entirely sure what the original `shake` is meant to do, would it be okay to retain 'shake' and add `movement` as an additional action?